src/Routes/seasons.ts

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * Seasons controller
+ */
+//import * as assert from "assert";
+import * as express from "express";
```

Comment:
pro tip: you can get VS Code to sort your imports using F1 -> Organize Imports

Reply:
Done.